collection_requirements.rst

```diff
@@ -384,6 +375,47 @@ Besides all the requirements listed in the `Development conventions <https://doc
   create new ``_info`` or ``_facts`` modules instead (for more information, refer to the `Developing modules guidelines <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_general.html#creating-an-info-or-a-facts-module>`_)
 * ``check_mode`` is supported in all ``*_info`` and ``*_facts`` modules (for more information, refer to the `Development conventions <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html#following-ansible-conventions>`_)
 
+.. _collection_dependencies:
+
+Collection Dependencies
+=======================
+
+**Notation:** if foo.bar has a dependency on baz.bam, we say that baz.bam is the collection *dependend on*, and foo.bar is the *dependent collection*.
```

Comment:
Yep, "dependend" doesn't exist. I've rewritten this sentence and made another small change. Can you check whether this is better now?

Reply:
I would suggest using:
1. `dependency` instead of `depended on` as there are a lot of `dependency|ies` are used across the paragraph and `depended on` only once OR
2. change `dependency|ies` to `collections depended on` everywhere (though the option 1 feels a bit better to me.

Generally the combination of `depended on` and `dependent collection` confuses me a lot:) Makes me focusing each time to figure out which means what:) Can be fair for other readers, i guess

Reply:
The problem is that `dependency` is the relation between two collections, while `the collection dependend on` is one of the two collections involved. I skimmed over all uses of `dependenc*`, and I think its used correctly everywhere to indicate the actual dependency (as the relation), and not the dependend on collection.

Reply:
OK but it confuses me anyway, though maybe it's my personal feature